packages/artillery-plugin-publish-metrics/lib/open-telemetry/index.js

```diff
@@ -385,7 +394,10 @@ class OTelReporter {
   endScenarioSpan(engine) {
     return function (userContext, ee, next) {
       const span = userContext.vars[`__${engine}ScenarioSpan`];
-      span.end(Date.now());
+      if (!span._ended) {
```

Comment:
`._ended` seems to be a private option, so maybe best not to rely on it if we can avoid it. 

Looking at the code it seems there's also a `this.endTime` that they set (https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-sdk-trace-base/src/Span.ts#L78). Maybe we could use that instead?